README.adoc

```diff
@@ -165,6 +165,34 @@ TIP: `exec` method also supports you pass how many times do you want to execute
 
 You can see full example at: https://github.com/arquillian/arquillian-cube-q/tree/master/ftest-toxic
 
+==== Randomizing
+
+Some of the discrete values set in some toxic like `slowClose`, `bandwidth`, `timeout` or `slice` can be randomized using mathematical distributions.
```

Comment:
I would consider rewording this sentence a bit - double `some` sounds a bit odd.

> in toxics such as `slowClose`... 

might sound better?